move2kube/m2k.sw.yml

```diff
@@ -192,8 +196,11 @@ states:
            functionRef:
              refName: createNotification
              arguments:
-               title: '"Move2Kube workflow " + $WORKFLOW.instanceId + " success"'
-               message: '"Move2Kube workflow  " + $WORKFLOW.instanceId + " on workspace " + .workspaceId + " and project " + .projectId + " was successful; the output is available in the branch " + .targetBranch + " of your git repository "+ .repositoryURL'
-               origin: "Move2Kube Workflow"
-               topic: "Move2Kube Workflow"
+               recipients:
+                 type: 'broadcast'
+               payload:
+                 title: '"Move2Kube workflow " + $WORKFLOW.instanceId + " success"'
```

Comment:
the issue was there before, but just to be aligned with the previous message, pls change `success` to `succeeded`

Reply:
fixed